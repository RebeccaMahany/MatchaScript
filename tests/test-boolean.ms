function void testBoolean(bool x, bool y) {
  if (x == y) {
    print("X is equal to Y");
  } else {
    print("X is not equal to Y");
  }

  bool z = true;
  if (x && z) {
    print("X and Z are both true");
  } else if (x || z) {
    print("X or Z is true");
  } else {
    print("X and Z are both false");
  }
}

testBoolean(true, false);
