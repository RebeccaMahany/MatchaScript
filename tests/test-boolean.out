X is not equal to Y
X and Z are both true
